### ISSUE TITLE
Authentication Loop

### ISSUE THREAD

**Issue by konradbjk** [langflow-ai/langflow#3438]
### Bug Description

Once I upgraded to v1.0.15 my docker instalaltion (with postgres16) I have been unable to login to the langflow. There was some loop of authentication, that was ending with 400 error stating, that I need to enable AUTO_LOGIN. When giving incorrect credentials, the behaviour is different, it shows right away that the credentials are wrong.

Downgrading to 1.0.14 is a workaround

### Reproduction

1. pull 1.0.15 docker image
2. Fill in env file
3. spin it up 
4. try to login

### Expected behavior

I can login with my superuser credentials

### Who can help?

@anovazzi1 @Cristhianzl  @lucaseduoli  @igorrCarvalho 

### Operating System

Ubuntu 24 LTS on Azure VM

### Langflow Version

1.0.15

### Python Version

3.12

### Screenshot

My `.env`
```properties
POSTGRES_USER=...
POSTGRES_PASSWORD=...
POSTGRES_DB=langflow

DO_NOT_TRACK=true
LANGFLOW_AUTO_LOGIN=False
LANGFLOW_SUPERUSER=...
LANGFLOW_SUPERUSER_PASSWORD=...
LANGFLOW_STORE_ENVIRONMENT_VARIABLES=true
LANGFLOW_SECRET_KEY=...
LANGFLOW_DATABASE_URL=postgresql://...:...@postgres:5432/langflow
```

my compose file
```yml
services:
  langflow:
    image: langflowai/langflow:1.0.14
    ports:
      - "80:7860"
    depends_on:
      - postgres
    env_file: ".env"
    volumes:
      - langflow-data:/app/langflow
      - langflow-config:/app/config
    networks:
      - node-network

  postgres:
    image: postgres:16
    hostname: postgres
    env_file: ".env"
    volumes:
      - langflow-postgres:/var/lib/postgresql/data
    networks:
      - node-network

volumes:
  langflow-postgres:
  langflow-data:
  langflow-config:

networks:
  node-network:
    driver: bridge
 ```

**Comment by Cristhianzl** [comment]
Hi @konradbjk,

Could you please follow up on PR https://github.com/langflow-ai/langflow/pull/3471? This PR should resolve any issues related to login using Langflow on Docker.

The fix will be included in version 1.0.16 of Langflow, so if you're using pip, I recommend waiting for the new release before updating to see if the issue persists. Alternatively, you can fork our main branch and use it locally.

Thank you!

**Comment by carlosrcoelho** [comment]
#3471
